projects/go-lib/src/lib/components/go-button/go-button.component.scss

```diff
@@ -181,14 +181,14 @@ $button__background--loading: rgba($theme-light-border, .4);
   }
 }
 
-.go-button-group {
+::ng-deep .go-button-group {
```

Comment:
@patryk0605 I don't love that we're using `ng-deep` here, but I suppose it makes sense to keep these classes with the buttons.